InnerEye/ML/run_ml.py

```diff
@@ -454,7 +475,9 @@ def create_activation_maps(self) -> None:
             activation_maps.extract_activation_maps(self.innereye_config)  # type: ignore
             logging.info("Successfully extracted and saved activation maps")
 
-    def mount_or_download_dataset(self) -> Optional[Path]:
+    def mount_or_download_dataset(self, azure_dataset_id: Optional[str],
+                                  local_dataset: Optional[Path],
+                                  idx: int = 0) -> Optional[Path]:
```

Comment:
Please add docu for the new args and behaviour. And idx is not a great variable name